src/roles/common/templates/group_vars.all.j2

```diff
@@ -10,7 +10,13 @@ xmpp_tls: {{ common.xmpp_tls | default(True) }}
 openflow_tls: {{ common.openflow_tls | default(True) }}
 
 vsd_fallocate_size_gb: {{ common.vsd_fallocate_size_gb | default(285) }}
-vsc_fallocate_size_gb: {{ common.vsc_fallocate_size_gb | default(2) }}
+{% if common.vsc_fallocate_size_gb is defined %}
```

Comment:
This is fine, but all you had to do is change the '2' to a '0' on line 13 of the previous file. The `default()` filter basically says "if the variable isn't defined, use this value as a default." You can see that all the other `allocate` variables do the same thing. When the code one is writing diverges from similar code nearby, it makes one stop and think that maybe the approach is more complicated. That is the case here. A simple change is what is needed, not all of this conditional execution code.

Reply:
I made the required changes, and changed the default from 2 to 0 and removed the conditional execution of the code and tested it again.